<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSimulationWindow</class>
 <widget class="QMainWindow" name="CreateSimulationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(147, 147, 147);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="select_img">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>92</y>
      <width>141</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px outset;
background-color: rgb(194, 194, 194);
color : black;</string>
    </property>
    <property name="text">
     <string>Map image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_model">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>141</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px outset;
background-color: rgb(194, 194, 194);
color : black;</string>
    </property>
    <property name="text">
     <string>Elevation model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_simulation">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>640</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px outset;
background-color: rgb(194, 194, 194);
color : black;</string>
    </property>
    <property name="text">
     <string>Start simulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_to_menu">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>640</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px outset;
background-color: rgb(194, 194, 194);
color : black;</string>
    </property>
    <property name="text">
     <string>Back to menu</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>92</y>
      <width>865</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px outset;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>92</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="map_img_path">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: black; </string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="elevation_model_path">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: black; </string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="simulation_path">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: black; </string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="select_simulation">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>141</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 1px outset;
background-color: rgb(194, 194, 194);
color : black;</string>
    </property>
    <property name="text">
     <string> Path to simulation</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
